classes/02class/exercises/c02-network08/roperto/README.md

```diff
@@ -0,0 +1,39 @@
+# c02-network08
+
+## Commands Execution Output
+
+- Answer the following questions
+  - What is the difference between inbound x outbound?
+    ```
+    Inbound filters packets arriving in the network, outbound filters packets leaving the network.
+    ```
+    
+  - What is going to happen by default? Everything allowed or denied?
+    ```
+    The NACL automatically created will allow all traffic because it has an 'allow all' rule.
+    
+    If all rules were deleted, then the default behavior would be to deny all traffic (deny all default rule).
+    ```
+    
+
+- Commands for creating a NACL to block SSH connection comming from the network `172.16.200.0/24`?
+
+```shell script
+NACL=acl-0db01b37de932e6db
+
+aws ec2 create-network-acl-entry --rule-number 150 --cidr-block 172.16.200.0/24 --ingress --network-acl-id $NACL --protocol tcp --port-range From=22,To=22 --rule-action deny
```

Comment:
are you using the default nacl for this? If yes, the default one has the `100` rule allowing all which would make your rule not to match. Same for egress

Reply:
Yes I am, good catch

Changed them to to 50 so they trigger first.

Reply:
we *never* miss anything mate 😅